packages/react-query/src/shared/proxy/utilsProxy.ts

```diff
@@ -355,16 +356,27 @@ export function createReactQueryUtils<TRouter extends AnyRouter, TSSRContext>(
 ) {
   type CreateReactUtilsReturnType = CreateReactUtils<TRouter, TSSRContext>;
 
+  const clientProxy = createTRPCClientProxy(context.client);
```

Comment:
creates the proxy outside of the callback function so it's not recreated on every property access